src/services/submissionService.ts

```diff
@@ -377,6 +386,191 @@ function validateIdInput(id: string): Result<unknown> {
     return validationResults
 }
 
+function validateSubmissionSearchFilters(filters: gqlTypes.SubmissionSearchFilters): Result<unknown> {
+    const validateSearchResults: Result<unknown> = {
+        data: undefined,
+        hasErrors: false,
+        errors: []
+    }
+
+    if (filters.orderBy?.some(order => order.fieldToOrder === 'id')) {
```

Comment:
instead of conditional checking orderBy every time, move the null/empty check first. 

You should move the orderby into a sub-function to make this a bit cleaner. You can then return early if the `orderBy` is null/empty.